src/main/java/trivia/Board.java

```diff
@@ -0,0 +1,24 @@
+package trivia;
+
+import java.util.Arrays;
+import java.util.List;
+
+class Board {
+
+  private String[] positionToQuestionCategoryName;
+
+  private Board(List<String> questionCategoryNames) {
+    positionToQuestionCategoryName = new String[questionCategoryNames.size() * 3];
+    for (int i = 0; i < positionToQuestionCategoryName.length; i++) {
+      positionToQuestionCategoryName[i] = questionCategoryNames.get(i % questionCategoryNames.size());
```

Comment:
This is the logic that should be applied when populating board deck with categories

Reply:
I left a response in the previous related comment.